nifi-nar-bundles/nifi-protobuf-bundle/nifi-protobuf-services/src/main/resources/docs/org.apache.nifi.services.protobuf.ProtobufReader/additionalDetails.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html lang="en">
+    <!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+    <head>
+        <meta charset="utf-8"/>
+        <title>ProtobufReader</title>
+        <link rel="stylesheet" href="../../../../../css/component-usage.css" type="text/css"/>
+    </head>
+
+    <body>
+        <p>
+            The ProtobufReader Controller Service reads and parses a Protocol Buffers Message from binary format and creates a Record object.
+            The Controller Service must be configured with the same '.proto' file that was used for the Message encoding, and the fully qualified
+            name of the Message type including its package (e.g. mypackage.MyMessage). The Reader will always generate one record from the input
+            data which represents the provided Protocol Buffers Message type.
+            Further information about Protocol Buffers can be found here:
+            <a href="https://protobuf.dev/">protobuf.dev</a>
```

Comment:
@mark-bathori Thanks for adding these additional details. I did have one question as  see on this referenced site there is Protobuf versions 2 and 3. Does the reader you created support both versions or only one?